src/lib/components/glue/form/Multiselect.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+
+	type Option = {
+		value: string;
+		label: string;
+		canRemove: boolean;
+	};
+
+	export let options: Option[];
+	export let heading: string = '';
+	export let placeholder: string = '';
+	export let selectedOptions: Option[] = [];
+	export let showError: boolean = false;
+
+	let showDropdown = false;
+	let search = '';
+	let inputElement: HTMLInputElement;
+	let dropdownElement: HTMLDivElement;
+
+	function addOption(option: Option) {
+		if (!selectedOptions.some((o) => o.value === option.value)) {
+			selectedOptions = [...selectedOptions, option];
+		}
+		showDropdown = false;
+		search = '';
+	}
+
+	function removeOption(value: string) {
+		selectedOptions = selectedOptions.filter((o) => o.value !== value);
+	}
+
+	$: filteredOptions = options.filter(
+		(option: Option) =>
+			!selectedOptions.some((o) => o.value === option.value) &&
+			option.label.toLowerCase().includes(search.toLowerCase())
+	);
+
+	function handleClickOutside(event: Event) {
+		if (inputElement && dropdownElement) {
+			if (
+				!inputElement.contains(event.target as Node) &&
+				!dropdownElement.contains(event.target as Node)
+			) {
+				showDropdown = false;
+			}
+		}
+	}
+
+	onMount(() => {
+		document.addEventListener('mousedown', handleClickOutside);
+		return () => {
+			document.removeEventListener('mousedown', handleClickOutside);
+		};
+	});
+</script>
+
+<div class="relative flex w-full flex-col" on:keypress={(e) => e.stopPropagation()}>
+	<label class="label">
+		<span class="label-text text-base font-semibold">{`${heading}`}</span>
+	</label>
+	<div class="flex w-full max-w-full flex-row flex-wrap items-center justify-start gap-2">
+		{#each selectedOptions as options}
+			<span
+				class=" flex w-fit items-center justify-center rounded-md bg-zinc-200 px-2 text-sm font-medium">
+				{options.label}
+				<button
+					class={`ml-1 text-zinc-400 hover:text-zinc-600 ${options.canRemove && 'hidden'}`}
```

Comment:
instead of specific colors, we need to use `text-base-content` provided by daisy https://daisyui.com/docs/colors/ for light, dark mode compatibility

if you'd like to change the tone, you can change the opacity of the color like `text-base-content/80` 

![image](https://github.com/cornellwebdev-admin/glue3-webdev/assets/32003815/6194cf7c-d65d-47ec-b19a-6246835febab)